pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -47,7 +47,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog/v2"
 	"k8s.io/utils/ptr"
-
+	azureutils "sigs.k8s.io/cluster-api-provider-azure/util/azure"
```

Comment:
I would rather not introduce this new dependency, can you copy the util over instead please?